readme.txt

```diff
@@ -302,6 +302,10 @@ Pantheon Advanced Page Cache integrates with WordPress plugins, including:
 See [CONTRIBUTING.md](https://github.com/pantheon-systems/wp-saml-auth/blob/master/CONTRIBUTING.md) for information on contributing.
 
 == Changelog ==
+
+= 1.3.0-dev (April 13, 2023) =
+* Adds support for WordPress Multisite [[#228](https://github.com/pantheon-systems/pantheon-advanced-page-cache/pull/228)].
```

Comment:
In the PR Body you mentioned

> For users who already have this plugin installed on a WPMS, they will need to click the Clear Cache button on the settings page to receive the prefixed keys.

We might want to make this a little more visible in the .org README's release notes as well?

Reply:
Good point! Updated this note in [1ac079a](https://github.com/pantheon-systems/pantheon-advanced-page-cache/pull/228/commits/1ac079aa93b5265e99660dab719f69154cfa8cf2).